8/README.md

```diff
@@ -0,0 +1,166 @@
+
+
+## Tango Device Server specification
+
+ This specification formally document the Tango Device Server model. 
+ Runtime implementation of the Device Server model in conforming Tango implementations MUST follow this specification.
+
+### Goals
+
+ In a Tango Control System a Device Server is in charge of managing the life cycle and the communication protocol of one or several Tango Devices.
+ The Device Server can also offer different services to the clients as well as for the Device.
+
+Additionally, it aims to:
+
+* Provide a blackbox that registers the operations executed on every Device
+
+* Be usable as a Device
+
+### Use Cases
+
+The main use cases for Device Server are:
+
+* to manage several Devices in the same process. 
+
+* to share the same channel of communication
+
+* to provide the connection protocols for accessing the managed Device
+
+* to provide different services like logging
+
+* to serialise the operation per Device
+
+* to sign up or sign out to the central registry 
+
+
+
+## Specification
+
+The Device Server manages the communication with one or several Devices.  Additionally it enforces the compatibility of behaviour with the Tango Control System and offer different meta service.
+
+The Device Server SHALL implement the following specifications to control and monitor the Device:
+
+* A Device Server represents the process which manages one or several Devices
+
+* A Device Server SHALL manage at least one Device. 
+
+* A Device Server MAY manage Devices belonging to different Classes
+
+* A Device MUST NOT be managed by more than one Device Server
+
+* A Device Server instance reference SHALL be unique in a Tango Control system
+
+* The communication to the Device Server SHALL be done through a Tango Device so called Administration Device or Admin Device .
+
+
+The specification introduces as well the definition of the process which are used for the Admnistration Device Name:
+
+* A Server is a type of program which execute a Device Server
+
+* An Instance is an unique identifier as several processes of the same Server MAY exist in a Tango Control System
+
+* A Server Instance is finally what identify the actual process in the Tango Control System
+
+* A Server Instance CAN only execute one Device Server
+
+
+The name of the Administration Device SHALL follow this convention:
+```ABNF
+administration-device-name = dserver "/" server "/" instance
+dserver = "dserver" ; dserver is a reserved domain following the device-name convention.
+server = family
+instance = member
+```
+
+* The member part of the Administration Device Name SHALL be the Instance
+
+* The family part of the Administration Device Name SHALL be the Server name
+
+
+#### Device Server interface
+
+
+| Property Name | description |
+|---|---|
+| polling_threads_pool_conf | TODO |
+
+| Command Name | description |
+|---|---|
```

Comment:
This table seems to expect 3 columns.
If you want to add a description too, it should be with 4 columns (WIP I guess)